apache-mesos-mixin/README.md

```diff
@@ -0,0 +1,113 @@
+# Apache Mesos Mixin
+
+Apache Mesos mixin is a set of configurable Grafana dashboards and alerts.
+
+The Apache Mesos mixin contains the following dashboards:
+
+- Apache Mesos overview
+
+and the following alerts:
+
+- ApacheMesosHighMemoryUsage
+- ApacheMesosHighDiskUsage
+- ApacheMesosUnreachableTasks
+- ApacheMesosNoLeaderElected
+- ApacheMesosInactiveAgents
+
+## Apache Mesos Overview
+
+The Apache Mesos overview dashboard provides details on resource utilization, queue usage, registrar state, allocator usage and master and agent logs. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance.
+
+When running the Mesos master and agent process, specify the logs director using `--log_dir=/var/log/mesos/master/` and `--log_dir=/var/log/mesos/agent/` so logs can be found in the path `/var/log/mesos/master/*.log*` and `/var/log/mesos/agent/*.log*` when ingesting logs through the Promtail config..
+
+Logs from Apache Mesos are not enabled by default which will result in empty log panels if the following step is not performed. When running the Mesos master and agent process, specify the logs director using `--log_dir=` for both the master and agent. This path will differ depending on if you are running Linux or Windows but is required so that Promptail can ingest the logs.
+
+For master:
+`--log_dir=/var/log/mesos/master/` on Linux masters
+`–-log_dir=C:\Program Files\mesos\master\` on Windows masters
+
+For agents:
+`--log_dir=/var/log/mesos/agent/` on Linux agents
+`–-log_dir=C:\Program Files\mesos\agent\` on Windows agents
+
+Apache Mesos logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for master and agent logs ingested into your logs datasource, please also include the matching `job`, and `mesos_cluster` labels onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+scrape_configs:
+  - job_name: integrations/apache-mesos
+    static_configs:
+      - targets:
+        - localhost
+        labels:
+          job: integrations/apache-mesos
+          mesos_cluster: '<your-cluster-name>'
+          __path__: /var/log/mesos/master/*.log*
+      - targets:
+        - localhost
+        labels:
+          job: integrations/apache-mesos
+          mesos_cluster: '<your-cluster-name>'
+          __path__: /var/log/mesos/agent/*.log*
+```
+
+![Apache Mesos Overview Dashboard 1](TODO)
```

Comment:
TODO needs to be changed to actual screenshot link?

Reply:
I just submitted the assets over to be uploaded.